src/components/lib/posix/posix.c

```diff
@@ -337,6 +484,7 @@ syscall_emulation_setup(void)
 	libc_syscall_override((cos_syscall_t)cos_set_thread_area, __NR_set_thread_area);
 	libc_syscall_override((cos_syscall_t)cos_set_tid_address, __NR_set_tid_address);
 	libc_syscall_override((cos_syscall_t)cos_clone, __NR_clone);
+	libc_syscall_override((cos_syscall_t)cos_futex, __NR_futex);
```

Comment:
I feel like all the system call names should be `cos_syscall_xxxx`. what do you think?

Reply:
Maybe `syscall_XXX` would be better? But I honestly think the current names are fine.

Reply:
yeah, i'm not very sure either.
But my feeling is that having a namespace for `posix` api and it's internals would probably be better. `cos_` is a generic namespace for everything in composite.